pkg/internal/ibmblockcsi/ibmblockcsi.go

```diff
@@ -42,52 +42,65 @@ func (c *IBMBlockCSI) Unwrap() *csiv1.IBMBlockCSI {
 	return c.IBMBlockCSI
 }
 
-// GetAnnotations returns all the annotations to be set on all resources
-func (c *IBMBlockCSI) GetAnnotations() labels.Set {
+// GetLabels returns all the labels to be set on all resources
+func (c *IBMBlockCSI) GetLabels() labels.Set {
 	labels := labels.Set{
 		"app.kubernetes.io/name":       config.ProductName,
 		"app.kubernetes.io/instance":   c.Name,
-		"app.kubernetes.io/version":    csiversion.Version,
```

Comment:
r u sure?   i think its better to keep it as well, since its vanila k8s recommendation.